fineract-provider/src/main/java/org/apache/fineract/infrastructure/documentmanagement/contentrepository/ContentRepositoryUtils.java

```diff
@@ -55,9 +55,8 @@ public static ImageMIMEtype fromFileExtension(ImageFileExtension fileExtension)
                     return ImageMIMEtype.JPEG;
                 case PNG:
                     return ImageMIMEtype.PNG;
-                default:
-                    throw new IllegalArgumentException();
             }
+            throw new IllegalArgumentException();
```

Comment:
Same here - shouldn't we just remove the default case altogether, not add anything back?